README.md

```diff
@@ -1,46 +1,429 @@
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  founders: [
+    {
+      // the current user
+      address: accounts[0],
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: anyAddress,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: anyAddress,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    }
+  ]
+});
+```
+
+### Create a new DAO with schemes
+
+Create a new DAO with schemes by including a "schemes" array.  This will register the schemes with all default permissions and voting machines. See `SchemesConfig` and `SchemeConfig`.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    { name: "GlobalConstraintRegistrar" }
+  ]
+});
+```
+
+### Create a new DAO overriding the default voting machine
+
+Override the default voting machine by adding a "votingMachines" element.  This will override the default voting machine, either at the root level to apply to all schemes, or within each scheme element to override for the specific scheme. See `NewDaoVotingMachineConfig` and `SchemesConfig`.
+
+#### Default root-level override for all schemes
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votePerc: 45,
+    ownerVote:false
+  }
+});
 ```
 
-You can also override the default configuration settings with environment variables:
+#### Scheme-specific override
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    {
+      name: "GlobalConstraintRegistrar",
+      votingMachineParams: {
+        votePerc: 30,
+      }
+    }
+  ]
+});
+```
+
+### Create a new DAO with a non-Universal Controller
+
+Create a DAO using the DAOstack `Controller` contract by passing `false` for `universalController`.
+
 ```javascript
-gasLimit=7000000 node yourprojectindex.js
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  universalController: false
+});
 ```
 
+### Create a new DAO with a non-default DaoCreator scheme
+
+Create a DAO using an alternative `DaoCreator` scheme by passing its address for `daoCreatorScheme`.  The alternative scheme must implement the same interface as the Arc `DaoCreator` contract.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  daoCreatorScheme: anAddress
+});
+```
+
+### Get a previously-created DAO
```

Comment:
you should also document how to get all the DAOs that have been created by looking up the NewOrg event on the DaoCreator scheme

Reply:
I was planning to do that when we add such a function to DaoCreator or DAO.  At that point it will be really clean.  Are you OK with waiting until then?